java/src/main/java/com/google/crypto/tink/integration/android/AndroidKeysetManager.java

```diff
@@ -100,13 +103,13 @@ private AndroidKeysetManager(Builder builder) throws GeneralSecurityException, I
     reader = builder.reader;
     if (reader == null) {
       throw new IllegalArgumentException(
-          "need to specify where to read the keyset from with Builder#withSharedPref");
+          "need to specify where to read the keyset from with Builder#withSharedPref or Builder#withKeysetReaderWriter");
```

Comment:
Small nit, but can you modify this (and the instance below) to adhere to https://google.github.io/styleguide/javaguide.html#s4.4-column-limit

Essentially:
```suggestion
          "need to specify where to read the keyset from with Builder#withSharedPref or "
              + "Builder#withKeysetReaderWriter");
```

Reply:
Done.